src/components/onboarding/popular_artist_list.tsx

```diff
@@ -0,0 +1,109 @@
+import * as React from 'react';
+import * as Relay from 'react-relay';
+import styled from 'styled-components';
+
+import { artsyNetworkLayer } from '../../relay/config';
+import PopularArtistQueryConfig from '../../relay/queries/popular_artist';
+
+import colors from "../../assets/colors";
+import * as fonts from '../../assets/fonts';
+
+import Icon from "../icon";
+import Input from '../input';
+
+const Link = styled.a`
+  display: flex;
+  font-size: 14px;
+  color: black;
+  text-decoration: none;
+  text-transform: uppercase;
+  font-family: ${fonts.primary.fontFamily};
+  border-top: 1px solid #e5e5e5;
+  &:hover {
+    background-color: #f8f8f8;
+  }
+`
+
+const Avatar = styled.img`
+  padding: 10px 15px 10px 10px;
+`
+
+const FullWidthCol = styled.div`
+  flex: 1;
+  display: flex;
+  align-items: center;
+`
+
+const Col = styled.div`
+  display: flex;
+  align-items: center;
+`
+
+const OnboardingSearchBox = styled.div`
+  width: 450px;
+  margin: 0 auto 100px;
+  border-bottom: 1px solid #e5e5e5;
+`
+
+export class PopularArtists extends React.Component<RelayProps, null> {
+  render() {
+    const popularArtists = this.props.popular_artists.artists.map(artist =>
+      <Link href="#">
+        <Col>
+          <Avatar src={artist.image.cropped.url} width={50} height={50} />
+        </Col>
+        <FullWidthCol>
+          {artist.name}
+        </FullWidthCol>
+        <Col>
+          <Icon name="follow-circle" color="black" fontSize="39px"/>
+        </Col>
+      </Link>
+    )
+
+    return (
+      <OnboardingSearchBox>
+        <div style={{ marginBottom: "35px" }}>
+          <Input placeholder="Search artist name" leftView={<Icon name="search" color={colors.graySemibold} />} block />
+        </div>
+
+        {popularArtists}
+      </OnboardingSearchBox>
+    )
+  }
+}
+
+const PopularArtistsContainer = Relay.createContainer(PopularArtists, {
+  fragments: {
+    popular_artists: () => Relay.QL`
+      fragment on PopularArtists {
+        artists {
+          name
+          image {
+            cropped(width: 50, height: 50) {
+              url
+            }
+          }
+        }
+      }
+    `,
+  },
+})
+
+export default function() {
+  Relay.injectNetworkLayer(artsyNetworkLayer())
+  return <Relay.RootContainer Component={PopularArtistsContainer} route={new PopularArtistQueryConfig()} />
+}
```

Comment:
This shouldn’t be in this file, but instead be used in the places where you want to mount the tree, in this case in the story file.

Reply:
Export the result of `Relay.createContainer` instead.